src/base/bittorrent/peerinfo.h

```diff
@@ -98,8 +98,8 @@ namespace BitTorrent
         void calcRelevance(const TorrentHandle *torrent);
         void determineFlags();
 
-        lt::peer_info m_nativeInfo;
-        qreal m_relevance;
+        lt::peer_info m_nativeInfo = {};
```

Comment:
It's unnecessary for class type.

Reply:
reverted.

Reply:
I'm having second thoughts about it. `peer_info` is actually a struct type and IMO it needs to be initialized properly since we have an additional constructor.
https://www.libtorrent.org/include/libtorrent/peer_info.hpp

Reply:
Sorry, don't understand you exactly.
In C++ **struct** just declares class type with public scope by default.

Reply:
The values in `peer_info` will be indeterminate if you don't initialize it, isn't?

Reply:
Yes, you're right :+1: